src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
-| AngleCurvature    | value (maximum angle per step)                       |
-| RadiusCurvature   | value (minimal radius of curvature)                  |
-| MinimumLength     | value                                                |
-| MaximumLength     | value                                                |
-| IntegrationOrder  | integer                                              |
-| Unidirectional    | bool                                                 |
-
-RECOMMENDED to appear within the "Seeding" field when `TractographyClass` is `local`:
-
-| **Key name** | **Description**                                                                                                          |
-| ------------ | ------------------------------------------------------------------------------------------------------------------------ |
-| SourceType   | {`sphere`,`voxels`,`surface`,`odf`}                                                                                      |
-| Location     | values: If "`SourceType`" is "`sphere`", provide a 4-vector containin the XYZ location and radius in mm of the sphere    |
-| Name         | string: If "`SourceType`" is not "`sphere`", give the filename from which seeds were derived                             |
-| CountType    | {`global`,`local`} (i.e. does Count reflect a count per voxel / vertex, or a single count for the entire reconstruction) |
-| Count        | integer                                                                                                                  |
-
-Example:
+   1. Directions only
 
-```Text
-<pipeline_name>/
-    sub-<participant_label>/
-        dwi/
-            sub-01_desc-DET_tractography.trk
-            sub-01_desc-DET_tractography.json
-```
+      Each consecutive pair of image volumes encodes inclination /
+      azimuth pairs, with order & convention identical to that above
+      (equivalent to spherical coordinate with assumed unity distance from
+      origin).
 
-```JSON
-{
-    "TractographyClass": "local",
-    "TractographyMethod": "deterministic",
-    "Parameters": {
-        "StepSize": 0.5
-    }
-}
-```
+      Number of image volumes is equal to (2x*N*), where *N* is the maximum
+      number of discrete orientations in any voxel in the image.
 
-```Text
-<pipeline_name>/
-    sub-<participant_label>/
-        dwi/
-            sub-01_subset-shortfibers_tractography.trk
-            sub-01_subset-shortfibers_tractography.json
-```
+1. <a name="data-3vector">*3-Vectors*</a>:
+
+   4D image where data across volumes within each voxel encode one or
+   more discrete orientations using triplets of axis dot products.
+
+   This representation may be used in one of two ways:
+
+   1. Value per direction
+
+      Each 3-vector, once explicitly normalized, provides a direction
+      on the unit sphere; the *norm* of each 3-vector additionally encodes
+      the magnitude of some mode parameter, the nature of which MUST be

```

Comment:
Did you intend on writing `model` instead of `mode`?

Reply:
Fixed in 6608713.